### PR TITLE
Timer Page functionality & SSL Setup

### PR DESCRIPTION
**Pull request by ChandlerCabrera**
**Timer Page**
- Fixed Flickering Issue

To test (WINDOWS - Command Prompt):
1. Open a terminal in /frontend directory
2. run "set HTTPS=true", followed by "npm start"
3. The browser should pop-up with a "This site is unsafe warning"
4. Run the ssl_setup.bat script. This creates a cert/encrypted key for localhost
5. The homepage should pop up. There should be a lock icon beside the browser
image

-------
Django changes:
URLs:
/timer/?uname (GET: returns timer for user. POST: stops timer, need to pass action = "stop" in body )
/start_timer/id (id= machine id. POST only. need to pass uname (username) in body)

Model: User_uses_Model. New field init_Balance
tasks.py: Updated stop time function to timer specification.

-------
Fixed MachineInfo.js

### CODE REVIEW

**Review by digengill — APPROVED** [review]

File: project/frontend/ssl_setup.bat [hunk 1]
```diff
@@ -0,0 +1,4 @@
+choco install mkcert
+mkcert -install
+mkcert localhost
+set HTTPS=true&&set SSL_CRT_FILE=localhost.pem&&set SSL_KEY_FILE=localhost-key.pem&&npm start
```
Comment (erick576) [inline project/frontend/ssl_setup.bat:4]: What exactly does this line set in my env variables? @ChandlerCabrera I'm getting an error after running this script saying 
> "You specified SSL_CRT_FILE in your env, but the file "C:\Users\Ericn\Documents\GitHub\capstone-project\project\frontend\localhost.pem" can't be found."

Comment (ChandlerCabrera) [reply project/frontend/ssl_setup.bat:4]: The *mkcert localhost* command should be creating a localhost.pem file in the frontend directory. Is that file missing for you?

Comment (ChandlerCabrera) [reply project/frontend/ssl_setup.bat:4]: That line enables HTTPs, specifies the certification file as localhost.pem (should be located in frontend folder), and specifies the key file as localhost-key.pem

Comment (erick576) [reply project/frontend/ssl_setup.bat:4]: Yes, Im not having any created when it runs that script. (I'm testing it in windows) @digengill did you run into any similar issues when you tried the SSL script?

Comment (ChandlerCabrera) [reply project/frontend/ssl_setup.bat:4]: I will try this setup again from scratch and see if I run into any issues

**Pull request closed without merging**